rest-service/manager_rest/storage/resource_models.py

```diff
@@ -565,14 +565,14 @@ class Node(SQLResourceBase):
         v2=['max_number_of_instances', 'min_number_of_instances']
     )
 
-    deploy_number_of_instances = db.Column(db.Integer, nullable=False)
     # TODO: This probably should be a foreign key, but there's no guarantee
     # in the code, currently, that the host will be created beforehand
     host_id = db.Column(db.Text)
-    max_number_of_instances = db.Column(db.Integer, nullable=False)
-    min_number_of_instances = db.Column(db.Integer, nullable=False)
-    number_of_instances = db.Column(db.Integer, nullable=False)
-    planned_number_of_instances = db.Column(db.Integer, nullable=False)
+    deploy_number_of_instances = db.Column(JSONString, nullable=False)
```

Comment:
Probably put a remark here that the columns have to be strings because of potential get_inputs?

(but then does the number_of_instances and planned_number_of_instances ever manage to not be an int?)